manifests/config.pp

```diff
@@ -4,7 +4,9 @@
   $solr_archive_file_name,
   $init_config,
   $solr_user,
-  $solr_port
+  $solr_port,
+  $gc_log_file_count = 5,
```

Comment:
that should not have defaults, because the defaults are already taken into account in the init.pp, right?